### PR TITLE
chore: github action to run prettier

### PR DESCRIPTION
**Pull request by pkra**
A first test

### CODE REVIEW

**Comment by pkra** [comment]
@jnurthen I thought we'd try a smaller spec first.

**Comment by pkra** [comment]
Some notes 

* graphics2.html had an unclosed section, throwing prettier off; I've temporarily fixed it
* I've set prettier's print-width to 200 which seemed decent. (I don't think we can set it to `Number.POSITIVE_INFINITY` via the CLI options -- but we could go 9999 or some such if people prefer.)

File: .github/workflows/prettier.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # Make sure the actual branch is checked out when running on pull requests
+          ref: ${{ github.head_ref }}
+          # This is important to fetch the changes to the previous commit
+          fetch-depth: 0
+
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.3
```
Comment (nschonni) [inline .github/workflows/prettier.yaml:23]: May want to pair this with a Dependabot config like https://github.com/w3c/aria-practices/blob/83d1329687a9f3e35737ff1a49bb2678eab02c15/.github/dependabot.yml#L11 to keep this Action up-to-date

Comment (pkra) [reply .github/workflows/prettier.yaml:23]: Thanks, @nschonni. I've made a note. Let's see if this actually makes it first though :smile:

**Comment by pkra** [comment]
@jnurthen would you take another look?

**Comment by pkra** [comment]
Overcome by events.

**Pull request closed without merging**